src/main/java/com/powsybl/openloadflow/network/LfBranch.java

```diff
@@ -101,24 +101,28 @@ default List<LfLimit> getLimits2(LimitType type) {
         return Collections.emptyList();
     }
 
-    boolean hasPhaseControlCapability();
-
-    Optional<DiscretePhaseControl> getDiscretePhaseControl();
-
     void updateState(LfNetworkStateUpdateParameters parameters);
 
     void updateFlows(double p1, double q1, double p2, double q2);
 
+    // phase control
+
+    boolean hasPhaseControlCapability();
```

Comment:
To be more coherent with `hasGeneratorVoltageControllerCapability`, I think it could be nice to call that method `hasPhaseControllerCapability` or something similar.

Reply:
Done